docs/roadmap.md

```diff
@@ -4,6 +4,149 @@ title: Babel Roadmap
+
+Most of the remaining tasks are in the [tracking issue](https://github.com/babel/babel/issues/10746), but there are still a few blockers:
+- We want to drop support for [Node.js 10](https://github.com/nodejs/Release), which stops being maintained on 2021-04-30;
+- We would like to release Babel as a pure ESM package. We are now in the process of converting our sources to be compatible with Node.js' ESM implementation, and while doing it we are examining how we can make it easier for people that currently use Babel to compile ESM to CJS;
+- We are trying to align our TypeScript AST with the `typescript-eslint` project. Our ASTs are _almost_ identical, but we need to introduce some small breaking changes to fully align;
+- Our release infrastructure doesn't support yet pre-releases, or using multiple "main" branches (one for Babel 8 and one for Babel 7);
+- We haven't figured out yet a policy for Babel 7 maintainance.
+
+### Implement new TC39 proposals
+
+Babel can currently parse all the Stage 3 proposals, and we can transform all of them except for top-level await, import assertions and JSON modules (which are best handled by bundlers working with the dependencies graph).
+
+We support all the Stage 2 proposals except for:
+- The new iteration of the decorators proposal (we need to implement both parsing and transform);
+- Transform for the Module Blocks proposal (we implemented parsing in Babel 7.13.0).
+
+We will implement support for decorators, and investigate if and how we can implement transform for module blocks.
+
+While we don't support many Stage 1 proposals, there have been recent updates to the pipeline operator and to do expressions: since we already support those proposals and the community is quite excited about them, we will update our implementations.
+
+There are also other proposal, such as pattern matching, that we didn't implement yet because their champions expect to do significant changes to the syntax and semantics. However, we are closely following their development to implement them in Babel as soon as they stabilize a bit.
+
+### Move `@babel/preset-env` into `@babel/core`
+
+A minimal Babel transforming setup requires at least three packages:
+- `@babel/core`
+- `@babel/preset-env`
+- a Babel "runner" (`@babel/cli`, `babel-loader`, `@rollup/plugin-babel`, etc)
+
+Moving `@babel/preset-env` directly into `@babel/core` has two big advantages:
+- It will be easier to configure Babel in simple projects, you would only need to enable a `compileJS: true` option in `babel.config.json` (or it could even be the default in the future -- it can't be default as `@babel/eslint-parser` does not compile the source)
+- It will make sure that the plugin versions are in sync with the `@babel/core` version, avoiding most of the bugs casued by accidentally incompatible packages versions
```

Comment:
```suggestion
- It will make sure that the plugin versions are in sync with the `@babel/core` version, avoiding most of the bugs caused by mismatched/incompatible packages versions
```